src/main/java/com/avispl/symphony/dal/communicator/lg/lcd/LgLCDDevice.java

```diff
@@ -114,55 +186,330 @@ public void controlProperties(List<ControllableProperty> controllableProperties)
 	}
 
 	/**
-	 * This method is recalled by Symphony to get the list of statistics to be displayed
+	 * This method is recalled by Symphony to control specific property
 	 *
-	 * @return List<Statistics> This return the list of statistics.
+	 * @param controllableProperty This is the property to be controlled
 	 */
 	@Override
-	public List<Statistics> getMultipleStatistics() throws Exception {
-		ExtendedStatistics extendedStatistics = new ExtendedStatistics();
+	public void controlProperty(ControllableProperty controllableProperty) throws Exception {
 
-		//controllable statistics
-		Map<String, String> controllable = new HashMap<String, String>() {{
-			put(controlProperties.power.name(), "Toggle");
-		}};
+		reentrantLock.lock();
+		try {
+			if (this.localExtendedStatistics == null) {
+				return;
+			}
+			Map<String, String> stats = this.localExtendedStatistics.getStatistics();
+			List<AdvancedControllableProperty> advancedControllableProperties = this.localExtendedStatistics.getControllableProperties();
+			isEmergencyDelivery = true;
+			String value = String.valueOf(controllableProperty.getValue());
+			String property = controllableProperty.getProperty();
+			if (controllableProperty.getProperty().equalsIgnoreCase(controlProperties.power.name())) {
+				if (controllableProperty.getValue().toString().equals(String.valueOf(LgLCDConstants.NUMBER_ONE))) {
```

Comment:
We can refactor the code here value.equals(String.valueOf(LgLCDConstants.NUMBER_ONE)
Rename the NUMBER_ONE constant to ON_STATUS_CODE to make the constant more understandable.